src/Nri/Ui/SortableTable/V4.elm

```diff
@@ -230,6 +241,24 @@ custom config =
         , sorter = config.sorter
         , width = config.width
         , cellStyles = config.cellStyles
+        , hidden = False
+        }
+
+
+{-| Creates a placeholder column which will reserve the space for the column,
+this can be used when multiple tables have similar data and we want to keep
+the size consistent.
+-}
```

Comment:
Same comment re. documenting that this switches to `table-layout: fixed`. Or maybe we can just say `See <link to docs for Table.placeholderColumn>`?